src/simulations/action_translator.py

```diff
@@ -706,6 +762,42 @@ def _flex_store_actions(self, res, time_step):
 
         return store_actions, store_bool_flex
 
+    def _flex_q_car_actions(self, res, time_step):
+        """Compute the flexible battery reactive power action from the optimisation result."""
+        no_flex_actions = self._get_no_flex_actions('flexible_q_car_action')
```

Comment:
'`flexible_q_car_action`' does not seem to be a valid input for `_get_no_flex_actions `?

Reply:
never mind I think I got confused with this comment.